frontend_tests/test_generator.py

```diff
@@ -16,9 +16,13 @@
     DRIVER = "firefox"
 
 # Test matrix
-SCENARIOS = [scenario_full, scenario_short]
+SCENARIOS = [
+    scenario_full, scenario_short,
+    scenario_no_jupyter, scenario_no_dockerfile,
+    scenario_timeout
+]
 USERS = ["user1", "user2"]
-TIMEOUT = 250
+TIMEOUT = 60
```

Comment:
it was me, who set 250. those times I had a hypothesis that tests can really be that slow on travis. now they run in less than 30 seconds